### PR TITLE
115 group new page

### PR DESCRIPTION
**Pull request by spaceraccoon**
Styled it to look like the whiteboard drawing. Simpleform doesn't have a built in inline checkbox for some reason (i.e. checkbox and labels on the same row vs. each option on a new line), so had to use a css override. Essentially, simpleform places each new option in a new "checkbox" div, which occupies a whole line. So I overrode this by making the checkbox css inline and 20% width.

### CODE REVIEW

**Comment by spaceraccoon** [comment]
Passes all checks and can be pulled!

**Comment by esoterik** [comment]
@spaceraccoon please add "needs review" after you update a PR

File: app/assets/stylesheets/inline_checkboxes_override.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.inline-checkbox {
+	display: inline-block;
+	width: 20%;
```
Comment (orenyk) [inline app/assets/stylesheets/inline_checkboxes_override.scss]: This totally breaks on mobile, we should have some kind of breakpoint here and probably just increase them to full width.

**Comment by orenyk** [comment]
We'll also have to rebase.

**Comment by spaceraccoon** [comment]
Tried rebasing but there are some weird conflicts? @esoterik @orenyk

**Comment by spaceraccoon** [comment]
@orenyk @esoterik rebased and ready for review!

**Pull request merged**